README.md

```diff
@@ -245,13 +243,14 @@ Then similarly with `matrix-react-sdk`:
 
 Finally, build and start Riot itself:
 
-1. `git clone git@github.com:vector-im/riot-web.git`
+1. `git clone https://github.com/vector-im/riot-web.git`
 1. `cd riot-web`
 1. `git checkout develop`
 1. `npm install`
 1. `rm -r node_modules/matrix-js-sdk; ln -s ../../matrix-js-sdk node_modules/`
 1. `rm -r node_modules/matrix-react-sdk; ln -s ../../matrix-react-sdk node_modules/`
-1. `npm start`
+1. `cp -vi config.sample.json config.json` # and configure it
+1. `npm start` # this should be run inside a `screen` or `tmux` session
```

Comment:
`npm start` does not need to be run in this way, I don't think this adds anything.